### PR TITLE
Portfolio webpage

### PR DESCRIPTION
**Pull request by kidde60**
### Portfolio webpage.
**On this webpage I:**

- added a README.md file.
- added an online version link.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,76 @@
...
+1. Set-up linters in your local env
+2. The npm package manager is going to create a node_modules directory to install all of your dependencies.You shouldn't commit that directory. To avoid that, you can create a .gitignore file and add node_modules to it:
+3. you need to initialize npm to create package.json file
+4. run npm install --save-dev hint@6.x -add hintrc -run npx hint .
+5. .stylelintrc
+6. run npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+7. run npx stylelint "**/*.{css,scss}"
+
+**Online version**
+- https://kidde60.github.io/
```
Comment (danmainah) [inline README.md:48]: - [ ] Kindly check on your link, it doesn't show your project, this is the screenshot of what am getting
image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @kidde60 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good PR and Description



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md (hunk 1 above)
Comment (kidde60) [reply README.md:48]: The project is on the branch **feature**

**Review by Ismailco — APPROVED** [review]
### Hi @kidde60,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally
✔️ Deploy Github Pages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**